dashboard/src/components/Tabs/Builds/StatusCard.tsx

```diff
@@ -0,0 +1,64 @@
+import { memo } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import type { ITreeDetails } from '@/pages/TreeDetails/TreeDetails';
+
+import BaseCard from '../../Cards/BaseCard';
+import StatusChartMemoized, { Colors } from '../../StatusChart/StatusCharts';
+
+interface IStatusCard<T> {
+  buildsSummary?: ITreeDetails['buildsSummary'];
+  toggleFilterBySection: (value: string, filterSection: T) => void;
+}
+
+const StatusCard = <T,>({
+  buildsSummary,
+  toggleFilterBySection,
+}: IStatusCard<T>): JSX.Element => {
+  const { formatMessage } = useIntl();
+  if (!buildsSummary) return <></>;
+
+  return (
+    <BaseCard
+      title={formatMessage({ id: 'buildTab.buildStatus' })}
+      content={
+        <StatusChartMemoized
+          type="chart"
+          pieCentralLabel={formatMessage({ id: 'global.executed' })}
+          pieCentralDescription={
+            <>
+              {(buildsSummary.invalid ?? 0) +
+                (buildsSummary.valid ?? 0) +
+                (buildsSummary.null ?? 0)}
```

Comment:
nit: create a `const invalid = buildsSummary.invalid ?? 0` so we handle the null situation only once